### PR TITLE
new(mp) Versa Packs Monitoring Procedure

### PR DESCRIPTION
**Pull request by pkriko**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+| device.sessions.failed.count      | Number of failed sessions on the device     | Count |
+| device.sessions.failed.percentage | Percentage of failed sessions on the device | %     |
+
+<!--Interfaces-->
+
+| Metric name                               | Description                                            | Unit |
+|:----------------------------------------- |:------------------------------------------------------ |:---- |
+| status                                    | Status of the interface                                |      |
+| interface.traffic.in.bitspersecond        | Incoming traffic going through the interface           | B/s  |
+| interface.traffic.out.bitspersecond       | Outgoing traffic going through the interface           | B/s  |
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:65]: b/s

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+<!--Ipsec-->
+
+| Metric name                      | Description                                        | Unit  |
+|:-------------------------------- |:-------------------------------------------------- |:----- |
+| ipsec.packets.in.count           | Number of incoming packets trough the IPsec tunnel | Count |
+| ipsec.traffic.in.bytespersecond  | Incoming traffic going through the IPsec tunnel    | B/s   |
+| ipsec.packets.out.count          | Number of outgoing packets trough the IPsec tunnel | Count |
+| ipsec.packets.out.count          | Number of outgoing packets trough the IPsec tunnel | Count |
+| ipsec.traffic.out.bytespersecond | Outcoming taffic going through the IPsec tunnel    | Count |
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:79]: B/s

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+| Metric name                                 | Description                               | Unit  |
+|:------------------------------------------- |:----------------------------------------- |:----  |
+| qos.policy.hit.count                        | Number of hits by QoS policy              | Count |
+| qos.policy.sessions.deny.count              | Number of sessions denied by QoS Policy   | Count |
+| qos.policy.packets.dropped.count            | Number of packets dropped by Qos Policy   | Count |
+| qos.policy.traffic.dropped.bytespersecond   | Traffic dropped by Qos Policy             | B/s   |
+| qos.policy.packets.forwarded.count          | Number of packets forwarded by Qos Policy | Count |
+| qos.policy.traffic.forwarded.bytespersecond | Traffic forwarded by QoS Policy           | B/s   |
+
+* par Application Qos Policy
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:97]:
```suggestion
* Par application Qos Policy

```

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+| sdwan.policy.packets.out.count          | Number of outgoing packets by SDWAN policy      | Count |
+| sdwan.policy.traffic.out.bytespersecond | Outcoming traffic going through by SDWAN Policy | B/s   |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+### Configuration SNMP de l'équipement
+
+La documentation officielle Versa Networks détaille les étapes pour activer et configurer le service SNMP.
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:123]: URL link ;-)

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+### Comment puis-je tester le Plugin et que signifient les options des commandes ?
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande
+depuis un collecteur Centreon en vous connectant avec l'utilisateur *centreon-engine*:
+
+```bash
+/usr/lib/centreon/plugins/centreon_versa_snmp.pl \
+  --plugin network::versa::snmp::plugin \
+  --mode devices \
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:189]:
```suggestion
  --plugin=network::versa::snmp::plugin \
  --mode=devices \

```

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+```bash
+OK: Device '0' cpu load: 8.00 %, memory used: 10.00%, sessions active: 0 (1000000), sessions failed: 0 (1000000) |
+'0#device.cpu.utilization.percentage'=8.00%;;;0;100 '0#device.memory.usage.percentage'=10.00%;;;0;100 
+'0#device.sessions.active.count'=0;;;0;1000000 '0#device.sessions.active.percentage'=0.00%;0:80;0:90;0;100
+'0#device.sessions.failed.count'=0;;;0;1000000 '0#device.sessions.active.percentage'=0.00%;;;0;100
+Device '0' cpu load: 8.00 %, memory used: 10.00%, sessions active: 0 (1000000), sessions failed: 0 (1000000)
+```
+
+Dans cet exemple, le Plugin récupère l'utilisation d'un équipement Versa Networks (```--plugin network::versa::snmp::plugin --mode devices```)
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:208]:
```suggestion
Dans cet exemple, le Plugin récupère l'utilisation d'un équipement Versa Networks (```--plugin=network::versa::snmp::plugin --mode=devices```)

```

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+identifié par l'adresse IP *10.0.0.1* (```--hostname=10.0.0.1```). Les paramètres de communauté et de version SNMP (```--snmp-version='2c' --snmp-community='mysnmpcommunity'```) 
+correspondants sont renseignés afin de pouvoir joindre l'équipement.
+
+Une alarme WARNING sera ainsi déclenchée si le pourcentage d'utilisation des sessions actives est supérieur à 80% (```--warning-sessions-active-prct='80'```);
+L'alarme sera de type CRITICAL au delà de 90% de cette même utilisation (```--critical-sessions-active-prct='90'```).
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:213]:
```suggestion
l'alarme sera de type CRITICAL au delà de 90% de cette même utilisation (```--critical-sessions-active-prct='90'```).

```

File: fr/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+
+Pour chaque mode, la liste de toutes les métriques, seuils associés et options complémentaires peut être affichée 
+en ajoutant le paramètre ```--help``` à la commande :
+
+```bash
+/usr/lib/centreon/plugins/centreon_versa_snmp.pl --plugin network::versa::snmp::plugin --mode devices --help
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-snmp.md:219]:
```suggestion
/usr/lib/centreon/plugins/centreon_versa_snmp.pl --plugin=network::versa::snmp::plugin --mode=devices --help

```

File: en/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+*community* and *version* are specified (```--snmp-version='2c' --snmp-community='mysnmpcommunity'```).
+
+This command would trigger a WARNING alarm if the active sessions used raise over 80% of the device session capacity 
+(```--warning-sessions-active-prct='80'```) and a CRITICAL alarm over 90% (```--critical-sessions-active-prct='90'```).
+
+For each Plugin mode, all the options as well as all the available thresholds can be displayed by adding the ```--help```
+parameter to the command:
+
+```bash
+/usr/lib/centreon/plugins/centreon_versa_snmp.pl --plugin network::versa::snmp::plugin --mode devices --help
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/network-versa-snmp.md:219]:
```suggestion
/usr/lib/centreon/plugins/centreon_versa_snmp.pl --plugin=network::versa::snmp::plugin --mode=devices --help

```

**Review by thibaults-centreon — CHANGES_REQUESTED** [review]

**Review by Sims24 — COMMENTED** [review]
Thanks for the great work. 

Few questions: 
- Any URL link available to explain how to configure SNMP? (as requested by @thibaults-centreon) 
- Why alphabetical order isn't keep in the sidebar ? 
- Don't you think "By monitor" would be more understandable that "by health monitor" ?

**Comment by Sims24** [comment]
Metrics have changed recently, I'll update the PR accordingly.

**Review by Sims24 — APPROVED** [review]

File: en/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+title: Versa Director Restapi
+---
+
+## Overview
+
+Versa Director provides the management, monitoring and orchestration capabilities
+needed to deliver Versa’s VNF-based network and security services.
+
+The Centreon Plugin-Pack relies on the Versa Director API to query and
+collect status and metrics of the Versa equipements managed by the Director.
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/network-versa-director-restapi.md:12]:
```suggestion
collect status and metrics of the Versa equipments managed by the Director.

```

File: en/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 11]
```diff
@@ -0,0 +1,246 @@
...
+| disk.usage.bytes                                   | Disk usage on the device                   | B     |
+| disk.free.bytes                                    | Free disk space on the device              | B     |
+| disk.usage.percentage                              | Percentage of disk usage on the device     | %     |
+| alarms.critical.count                              | Number of critical alarms on the device    | Count |
+| alarms.major.count                                 | Number of major alarms on the device       | Count |
+| alarms.minor.count                                 | Number of minor alarms on the device       | Count |
+| alarms.warning.count                               | Number of warning alarms on the device     | Count |
+| alarms.inderminate.count                           | Number of inderminate alarm on the device  | Count |
+| policy.violation.packets.dropped.novalidlink.count | Number of packets dropped by no valid link | Count |
+| policy.violation.packets.dropped.slaaction.count   | Number of packets dropped by sla action    | Count |
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/network-versa-director-restapi.md:58]:
```suggestion
| Metric name                                        | Description                                 | Unit  |
|:-------------------------------------------------- |:------------------------------------------- |:----- |
| status                                             | Status of the device                        |       |
| memory.usage.bytes                                 | Memory usage on the device                  | B     |
| memory.free.bytes                                  | Free memory on the device                   | B     |
| memory.usage.percentage                            | Percentage of memory usage on the device    | %     |
| disk.usage.bytes                                   | Disk usage on the device                    | B     |
| disk.free.bytes                                    | Free disk space on the device               | B     |
| disk.usage.percentage                              | Percentage of disk usage on the device      | %     |
| alarms.critical.count                              | Number of critical alarms on the device     | Count |
| alarms.major.count                                 | Number of major alarms on the device        | Count |
| alarms.minor.count                                 | Number of minor alarms on the device        | Count |
| alarms.warning.count                               | Number of warning alarms on the device      | Count |
| alarms.indeterminate.count                         | Number of indeterminate alarm on the device | Count |
| policy.violation.packets.dropped.novalidlink.count | Number of packets dropped by no valid link  | Count |
| policy.violation.packets.dropped.slaaction.count   | Number of packets dropped by sla action     | Count |

```

File: en/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+id: network-versa-snmp
+title: Versa SNMP
+---
+
+## Overview
+
+Versa Networks provides secure cloud IP architecture. 
+In particular, SD-WAN that aims to phase out conventional WAN infrastructures.
+
+The Centreon Plugin-Pack relies on the SNMP protocol to query and collect status and metrics of the Versa equipements.
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/network-versa-snmp.md:11]:
```suggestion
The Centreon Plugin-Pack relies on the SNMP protocol to query and collect status and metrics of the Versa equipments.

```

File: en/integrations/plugin-packs/procedures/network-versa-snmp.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+| sdwan.policy.traffic.out.bytespersecond | Outcoming traffic going through by SDWAN Policy | B/s   |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+### Device Configuration
+
+The SNMP agent must be configured and running on the Versa Networks device.
+Please refer to the manufacturer documentation to achieve this
```
Comment (thibaults-centreon) [inline en/integrations/plugin-packs/procedures/network-versa-snmp.md:121]:
```suggestion
Please refer to the manufacturer documentation to achieve this.

```

File: fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 14]
```diff
@@ -0,0 +1,241 @@
...
+| disk.usage.bytes                                   | Disk usage on the device                   | B     |
+| disk.free.bytes                                    | Free disk space on the device              | B     |
+| disk.usage.percentage                              | Percentage of disk usage on the device     | %     |
+| alarms.critical.count                              | Number of critical alarms on the device    | Count |
+| alarms.major.count                                 | Number of major alarms on the device       | Count |
+| alarms.minor.count                                 | Number of minor alarms on the device       | Count |
+| alarms.warning.count                               | Number of warning alarms on the device     | Count |
+| alarms.inderminate.count                           | Number of inderminate alarm on the device  | Count |
+| policy.violation.packets.dropped.novalidlink.count | Number of packets dropped by no valid link | Count |
+| policy.violation.packets.dropped.slaaction.count   | Number of packets dropped by sla action    | Count |
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md:58]:
```suggestion
| Metric name                                        | Description                                 | Unit  |
|:-------------------------------------------------- |:------------------------------------------- |:----- |
| status                                             | Status of the device                        |       |
| memory.usage.bytes                                 | Memory usage on the device                  | B     |
| memory.free.bytes                                  | Free memory on the device                   | B     |
| memory.usage.percentage                            | Percentage of memory usage on the device    | %     |
| disk.usage.bytes                                   | Disk usage on the device                    | B     |
| disk.free.bytes                                    | Free disk space on the device               | B     |
| disk.usage.percentage                              | Percentage of disk usage on the device      | %     |
| alarms.critical.count                              | Number of critical alarms on the device     | Count |
| alarms.major.count                                 | Number of major alarms on the device        | Count |
| alarms.minor.count                                 | Number of minor alarms on the device        | Count |
| alarms.warning.count                               | Number of warning alarms on the device      | Count |
| alarms.indeterminate.count                         | Number of indeterminate alarm on the device | Count |
| policy.violation.packets.dropped.novalidlink.count | Number of packets dropped by no valid link  | Count |
| policy.violation.packets.dropped.slaaction.count   | Number of packets dropped by sla action     | Count |

```

File: fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 15]
```diff
@@ -0,0 +1,241 @@
...
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Installer le Plugin sur l'ensemble des collecteurs Centreon supervisant
+des équipements Versa via l'API du Versa Director :
+
+```bash
+yum install centreon-plugin-Network-Versa-Director-Restapi
+```
+
+2. Installer le Plugin-Pack 'Versa Director Restapi' depuis la page "Configuration > Plugin packs > Manager" sur l'interface Web de Centreon.
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md:94]:
```suggestion
2. Installer le Plugin-Pack *Versa Director Restapi* depuis la page "Configuration > Plugin packs > Manager" sur l'interface Web de Centreon.

```

File: fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+yum install centreon-plugin-Network-Versa-Director-Restapi
+```
+
+2. Installer le RPM du Plugin-Pack contenant les modèles de supervision:
+
+```bash
+yum install centreon-pack-network-versa-director-restapi
+```
+
+3. Installer le Plugin-Pack 'Versa Director Restapi' depuis la page "Configuration > Plugin packs > Manager" sur l'interface Web de Centreon.
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md:111]:
```suggestion
3. Installer le Plugin-Pack *Versa Director Restapi* depuis la page "Configuration > Plugin packs > Manager" sur l'interface Web de Centreon.

```

File: fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 17]
```diff
@@ -0,0 +1,241 @@
...
+|           | PROXYURL                | Proxy URL. (eg. http://myproxy.int:3128)                                   |
+
+> Utiliez le module de découverte pour ajouter à votre supervision vos équipements Versa.
+> Allez dans le menu "Configuration > Host > Discovery" et utilisez le provider *Versa Networks devices (Director RestAPI)*
+
+## FAQ
+
+### Comment tester un contrôle en ligne de commande et que signifient les options principales ?
+
+A partir du moment ou le Plugin est installé, vous pouvez tester directement celui-ci en ligne de commande depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md:139]:
```suggestion
Une fois le Plugin installé, vous pouvez tester directement celui-ci en ligne de commande depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:

```

File: fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md [hunk 18]
```diff
@@ -0,0 +1,241 @@
...
+    health monitor 'physical ports' up: 0, down: 0, disabled: 0
+    health monitor 'reachability status' up: 1, down: 0, disabled: 0
+    health monitor 'service status' up: 1, down: 0, disabled: 0
+```
+
+Cette commande supervise un équipement Versa Networks **CENFRGW101** (```--filter-device-name='^CENFRGW101$'```)
+à l'aide l'API Versa Director (```--plugin=network::versa::director::restapi::plugin --mode=devices```).
+Le plugin requête l'API du Versa Director **10.0.0.1** (```--hostname=10.0.0.1```) avec l'utilisateur
+**jdoe** et son mot de passe (```--api-username='jdoe' --api-password='6fbadZEJbsLG'```).
+Un équipement Versa peut être lié à plusieurs organisations, nous utilisons une widlcard (```--organization='.*'```).
```
Comment (thibaults-centreon) [inline fr/integrations/plugin-packs/procedures/network-versa-director-restapi.md:202]:
```suggestion
Un équipement Versa peut être lié à plusieurs organisations, nous utilisons une *wildcard* (```--organization='.*'```).

```

**Review by thibaults-centreon — CHANGES_REQUESTED** [review]

**Review by thibaults-centreon — APPROVED** [review]

**Pull request merged**